app.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express');
+const Sequelize = require('sequelize');
+
+const app = express();
+const connection = new Sequelize({
+  database: 'ferretdb',
+  username: 'ferret',
+  password: null,
+  host: 'localhost',
+  dialect: 'postgres',
+  define: {
+    timestamps: false,
+  },
+});
+
+const User = connection.define('user', {
```

Comment:
As I said, this goes into `app/models/user.js`.

Reply:
I know I didn't say anything about this, but we are going to use DB migrations to handle database changes :)